options.ts

```diff
@@ -13,12 +13,12 @@ export class OptionType {
 export class OptionsBase {
 	private optionsWhiteList = ["ui", "recursive", "reporter", "require", "timeout", "_", "$0"]; // These options shouldn't be validated
 	public argv: IYargArgv;
-	private static GLOBAL_OPTIONS: IDictionary<IDashedOption> = {
+	private GLOBAL_OPTIONS: IDictionary<IDashedOption> = {
```

Comment:
Instead of setting this property to dynamic in favor of static, you could assign
`OptionsBase.GLOBAL_OPTIONS.profileDir.default = this.defaultProfileDir`
in the constructor.

This is only a minor suggestion of course as the benefit would be next to none so definitely not a merge-stopper